scripts/pipeline-launch/launch.sh

```diff
@@ -1052,6 +1052,36 @@ echo
 ######################################################
 
 
+######################################################
+echo "Create package manager restriction wrappers"
+echo "-"
+######################################################
+
+CP_USR_BIN="/usr/cpbin"
+
+mkdir -p "$CP_USR_BIN"
+IFS=',' read -r -a RESTRICTING_PACKAGE_MANAGERS <<< "$CP_RESTRICTING_PACKAGE_MANAGERS"
+
+for MANAGER in "${RESTRICTING_PACKAGE_MANAGERS[@]}"
+do
+    MANAGER_PATH=$(which "$MANAGER")
+    if [[ "$?" == 0 ]]
+    then
+        MANAGER_WRAPPER_PATH="$CP_USR_BIN/$MANAGER"
+        echo "$COMMON_REPO_DIR/shell/package_manager_restrictor \"$MANAGER_PATH\" \"\$@\"" > "$MANAGER_WRAPPER_PATH"
+        chmod +x "$MANAGER_WRAPPER_PATH"
```

Comment:
Is `+x` enough? I.e. don't we need to set any other permissions for non-root users?

Reply:
Probably, the most certain way to set permissions for the wrapper scripts is to use permissions of the original package manager binaries.

Done in 44f08c878f1637b5b845416f720565dbf31add13.